.github/workflows/ci.yml

```
@@ -26,6 +45,13 @@ jobs:
           npm install && npm run build
           git diff --quiet || (echo "::error file=grammar.js::Generated parser differs from the checked in version" && exit 1)
 
-      - name: Parser tests 
+      - name: Parser tests
         shell: bash
         run: npm install && npm run test
+
+      - name: Smoke test
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

```suggestion
        if: ${{ matrix.os == 'ubuntu-20.04' }}
```

Now that I think about it's probably easier to use `runner.os == "Linux"` if that's the right value https://docs.github.com/en/actions/learn-github-actions/environment-variables#detecting-the-operating-system